vcd/resource_vcd_inserted_media.go

```diff
@@ -0,0 +1,142 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/govcd"
+	"github.com/vmware/go-vcloud-director/types/v56"
+	"log"
+)
+
+func resourceVcdInsertEjectMedia() *schema.Resource {
```

Comment:
Thank you for the renames. Final ask is to rename this func resourceVcdInsertEjectMedia() -> resourceVcdInsertedMedia() to have consistency with all the other resources where we have the entry point naming mapped to resource naming.